eth-bridge/src/main/kotlin/jp/co/soramitsu/soranet/eth/bridge/ReferendumHashProofHandler.kt

```diff
@@ -91,7 +92,8 @@ class ReferendumHashProofHandler(
             null
     }
 
-    private fun createProof(hash: String): String {
+    private fun createProof(proof: String): String {
+        val hash = hashToProve(proof)
```

Comment:
Looks good. 
However this method may be refactored. We have the same block of code at line 139.
We can avoid such error in future with it.

Reply:
Where is 139 line? 👀 
There is 119 lines total.
btw I updated the fix.

Reply:
Check line 139 in jp.co.soramitsu.soranet.eth.bridge.WithdrawalProofHandler#createProof